CEXT-3242: Release the new version of commerce webhooks without admin UI (#213)
AdobeDocs/commerce-extensibility: src/pages/webhooks/release-notes.md

These release notes describe the latest version of Adobe Commerce Webhooks.

## Version 1.5.0

### Release date

May 15, 2024

### Changes

> * The Admin configuration UI for webhooks is no longer supported. This behavior is developer-oriented and gives an unprecedented amount of access to admins which can be dangerous in many cases.

Reviewer: ```suggestion
* The Admin configuration UI for webhooks is no longer supported. This behavior is developer-oriented and gives an unprecedented amount of access to admins, which could be inappropriate in some cases.
```
